src/config/Config.ts

```diff
@@ -127,6 +127,12 @@ export const sortedIntegrationList = (
 
 export const defaultIconList = {
   [IntegrationCategory.COMMUNICATIONS]: <IntegrationIconTypes>{
+    [UserIntegrationType.EMAIL]: <IntegrationIcon>{
+      name: IntegrationType.EMAIL_SUBSCRIPTION,
+      product_name: 'Email',
+      icon_url:
```

Comment:
Can we add email icon to frontend assets and use it from there?

cc: @epwinchell